src/main/java/com/SEtrack/Hotel/controllers/RoomController.java

```diff
@@ -1,57 +1,58 @@
 package com.SEtrack.Hotel.controllers;
 
 import com.SEtrack.Hotel.models.Room;
+import com.SEtrack.Hotel.repositories.RoomRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
 
 import java.util.ArrayList;
+import java.util.List;
 
 //This is the RoomController class. Has an ArrayList holding the rooms. Methods: Add, remove and update the room.
```

Comment:
voldoet nog niet aan de JavaDocs comments

Reply:
![image](https://user-images.githubusercontent.com/11330327/33134470-17f4ffa4-cfa0-11e7-91c0-95e04154c0bf.png)

Ik krijg ook nog een error bij het toevoegen van een kamer

Reply:
Als het goed is, is het nu gefixt. Wil je even laten weten of het goed is?